workflow/lambdas/process_data.py

```diff
@@ -1,24 +1,25 @@
-# TODO: Finish Process data lambda
-from json import dumps, loads
 import os
+import logging
+from json import dumps, loads
 
 import boto3
 from purpleair.network import SensorList
 
-
 s3 = boto3.resource("s3")
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
 
-# --------------- Start of s3 formatting functions ---------------
 
+# --------------- Start of s3 formatting functions ---------------
 def grab_ak_data(purple_air_data):
     ak_data = []
     for info in purple_air_data:
         try:
             if 52 <= info['lat'] <= 75 and -175 <= info['lon'] <= -128:
                 ak_data.append(info)
         except TypeError as e:
-            print(e)
+            logger.info(f"The falling data: {info}, gave the following error {e}")
```

Comment:
Should "falling" be "following"?

Reply:
Yes, idk what my brain was doing at that moment.